erpnext/public/js/hub/pages/Home.vue

```diff
@@ -58,6 +58,11 @@ export default {
 		this.search_value = '';
 		this.get_items();
 	},
+	mounted() {
+		frappe.route.on('change', () => {
+			this.get_items();
```

Comment:
This will be called on every route change. Shouldn't this be called only on the `Home` route? Add a check for it.